samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -87,55 +87,102 @@ public void writeWorkItems(
     }
   }
 
-  public boolean doWorkSingleTransaction(DatabaseClient dbClient, boolean doneValue) {
+  private boolean doWorkSingleTransactionLocking(DatabaseClient dbClient, boolean doneValue) {
```

Comment:
I think we can remove `doneValue`, and just use false. I included it so that as I was debugging this I could move the worklists back and forth between true and false without creating a new table, but that is unnecessary for the codelab.